packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -7,14 +7,15 @@ import { Initializable } from "@openzeppelin/contracts/proxy/utils/Initializable
 // Libraries
 import { GameType, OutputRoot, Claim, GameStatus, Hash } from "src/dispute/lib/Types.sol";
 import { Unauthorized } from "src/libraries/errors/CommonErrors.sol";
-import { UnregisteredGame, InvalidGameStatus } from "src/dispute/lib/Errors.sol";
+import { InvalidAnchorGame } from "src/dispute/lib/Errors.sol";
 
 // Interfaces
 import { ISemver } from "interfaces/universal/ISemver.sol";
 import { IFaultDisputeGame } from "interfaces/dispute/IFaultDisputeGame.sol";
 import { IDisputeGame } from "interfaces/dispute/IDisputeGame.sol";
 import { IDisputeGameFactory } from "interfaces/dispute/IDisputeGameFactory.sol";
 import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
+import { IOptimismPortal2 } from "interfaces/L1/IOptimismPortal2.sol";
 
```

Comment:
nit: we decided on the style of using named return args, but they are not used consistently throughout the codebase. We should generally refactor towards the style as we touch parts of the codebase and not add net new code that breaks the style. Thoughts here?

Reply:
Good point, we can add named return args here.